sites/header/header-docs-en.html

```diff
@@ -37,6 +37,92 @@ <h2>How to implement</h2>
 
 <p>This version requires the implementation of the <a href="../gcweb-menu/gcweb-menu-docs-en.html">gcweb-menu version 2.0</a> where the <code>&lt;div class="container"&gt;</code> has been relocated in the header so outside the "gcweb-menu" component.</p>
 
+<p>Here a code sample showing all the commonly included component where some might require some customization by your web content management system such the breadcrumb components.</p>
+
+<pre><code>&lt;nav>
+	&lt;ul id="wb-tphp">
+		&lt;li class="wb-slc">&lt;a class="wb-sl" href="#wb-cont">Skip to main content&lt;/a>&lt;/li>
+		&lt;li class="wb-slc visible-sm visible-md visible-lg">&lt;a class="wb-sl" href="#wb-info">Skip to About this site&lt;/a>&lt;/li>
+	&lt;/ul>
+&lt;/nav>
```

Comment:
@duboisp Do we consider the Skip links as being part of the Header? I was asking myself this question earlier. I'm comfortable with both answers, that said we'll need to have the documentation reflect this eventually.

Reply:
No, it is not in the site header because it is outside the `<header>` element scope. But I think it is fine to have it here in the code sample as the intend is to provide a quick way to insert all HTML that is before the `<main>` element. What do you think?